backend/flask_api.py

```diff
@@ -191,6 +192,62 @@ def delete_project(project_id):
         return make_response('Success')
 
 
+@app.route('/api/projects/<uuid:project_id>', methods=['PUT'])
+def update_project(project_id):
+    """Updates Project by ID
+
+    Args:
+        project_id, updated manifest in json/json5 format
+
+    Returns:
+        response: Success response
+                  or 404 if project is not found
+                  or 409 if project_id differs from manifestID
+                  or 500 in case of validation error
+    """
+    try:
+        res = coll.find_one({'_id': project_id})
+        if res is None:
+            raise ApiException("Project not found", 404)
+        elif request.is_json or request.headers['content-type'] == "application/json5":
+            if request.is_json:
+                manifest = request.json
+                if manifest['_id'] != str(project_id):
+                    raise ApiException("Updated project owns different id", 409)
+            else:
+                manifest = json5.loads(request.data.decode("utf-8"))
+                if '_id' in manifest:
+                    if manifest['_id'] != str(project_id):
```

Comment:
I did not do that because of the possibillity of users updating their local json/json5, they had before and give that one as input. Then they do not have the db id, but it could be a possible use case. It would also be  an update an the id has to be set.